tools/tuf/releaser.sh

```diff
@@ -132,19 +132,18 @@ release_fleetd_to_edge () {
     BRANCH_NAME="release-fleetd-v$VERSION"
     ORBIT_TAG="orbit-v$VERSION"
     if [[ "$SKIP_PR_AND_TAG_PUSH" != "1" ]]; then
```

Comment:
Maybe we can do the git tag here only.

And separately (later) make a PR with the changes against `main`?

Reply:
Ok -- I left it in for now with the option to turn it off, since it does seem convenient to automate the changelog for normal releases.  It now calls a new `create_fleetd_release_pr` function which can also be called separately via `ACTION=create-fleetd-release-pr ./tools/tuf/releaser.sh`